content/exporters/custom-exporter/Java/Metrics.md

```diff
@@ -0,0 +1,312 @@
+
+In this example, the metrics will be written to the log, which will out sent to the console when running from the command line
+
+```java
+StringBuffer sb = new StringBuffer();
+for (Point p : ts.getPoints()) {
+  Timestamp t = p.getTimestamp();
+  long s = t.getSeconds();
+  long nanos = t.getNanos();
+  String line = s + "\t" + nanos + "\t" + p.getValue();
+  sb.append(line);
+}
+logger.info("Timeseries to export:\n" + sb);
+```
+
+The [IntervalMetricReader](https://github.com/census-instrumentation/opencensus-java/blob/master/exporters/metrics/util/src/main/java/io/opencensus/exporter/metrics/util/IntervalMetricReader.java) class conveniently provides a service that runs in the background and writes out the metric values that have been received during the collection interval. It may be useful to keep a reference to it to shutdown the background thread gracefully.
+
+## Runnable example
+
+Following the previous description, here is a ful runnable example, that we'll run using Maven. Create the Java class in the file `src/main/java/io/opencensus/examples/exporter/stats/ExampleStatsExporter.java`.
+
+{{<tabs Java_Code pom_xml>}}
+{{<highlight java>}}
+package io.opencensus.examples.exporter.stats;
+
+import io.opencensus.common.Timestamp;
+import io.opencensus.exporter.metrics.util.IntervalMetricReader;
+import io.opencensus.exporter.metrics.util.MetricExporter;
+import io.opencensus.exporter.metrics.util.MetricReader;
+import io.opencensus.metrics.LabelKey;
+import io.opencensus.metrics.LabelValue;
+import io.opencensus.metrics.Metrics;
+import io.opencensus.metrics.export.Metric;
+import io.opencensus.metrics.export.MetricDescriptor;
+import io.opencensus.metrics.export.Point;
+import io.opencensus.metrics.export.TimeSeries;
+import io.opencensus.stats.Aggregation;
+import io.opencensus.stats.Aggregation.Distribution;
+import io.opencensus.stats.BucketBoundaries;
+import io.opencensus.stats.Measure.MeasureLong;
+import io.opencensus.stats.Stats;
+import io.opencensus.stats.StatsRecorder;
+import io.opencensus.stats.View;
+import io.opencensus.stats.View.Name;
+import io.opencensus.stats.ViewManager;
+import io.opencensus.tags.TagKey;
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.Collection;
+import java.util.List;
+import java.util.Random;
+import java.util.logging.Logger;
+
+/**                                                                                                                                         
+ * Example OpenCensus Stats Exporter.
+ *
+ * @since 0.20
```

Comment:
This is not needed since examples are not part of release artifacts.

Reply:
Removed in new push